src/blocking/mod.rs

```diff
@@ -52,7 +52,7 @@ impl SecretService<'_> {
     }
 
     /// Get all collections
-    pub fn get_all_collections(&self) -> Result<Vec<Collection>, Error> {
+    pub fn get_all_collections(&'_ self) -> Result<Vec<Collection<'_>>, Error> {
```

Comment:
Nit: Is it possible to make all of these at least be `&'a` if a combination of `&self` and `<'_>` won't work? I don't think I've ever seen `&'_ self` in the wild before.

Reply:
This was due to my failing to declare the 'a lifetime on the impl wrapper. Fixed.